settings/trex.toml

```diff
@@ -32,10 +32,17 @@ SELECT *
 FROM d_stops_cluster
 WHERE geom && !bbox!
 ORDER BY max_avg_distance
-LIMIT 1000
+LIMIT 999
```

Comment:
why 999 over 1000 ?

Reply:
T-rex emit a warning à 1000 objects. Since we deal with the limit by ordering the objects, the warning in the log have no more cense.